arch/arm64/core/reset.c

```diff
@@ -124,8 +124,12 @@ void z_arm64_el2_init(void)
 		SCTLR_SA_BIT);		/* Enable SP alignment check */
 	write_sctlr_el2(reg);
 
+#ifdef CONFIG_SOC_RK3568
+	reg = HCR_RW_BIT;		/* Cortex-a55 cpu should clear other bits */
+#else
```

Comment:
Not sure what is the problem here. If there is a general problem that must be fixed in a generic way.

Reply:
If not clear other bits, system will occur issue when `eret` from EL2 to EL1. I will try to fix it in more generic way.